src/db/schema.sql

```diff
@@ -0,0 +1,69 @@
+-- why is collection_id before id in BSO?
```

Comment:
It's just a convention to match the primary key order (collection_id, id); the order is important there because it allows `SELECT * FROM bso WHERE collection_id=X` to execute efficiently using a prefix of the primary key index.